sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -69,6 +69,7 @@ object Config {
     case class File(path: URI) extends Output
     case class PubSub(subscription: String) extends Output
     case class Kafka(brokers: String, topic: String, producerConf: Map[String, String] = Map.empty) extends Output
+    case class Http(endpoint: String) extends Output
```

Comment:
If you made endpoint a `org.http4s.Uri` instead of a `String` in the config, then you wouldn't need to handle parsing exceptions later on when generating requests.